src/xbot/common/controls/sensors/XboxControllerWpiAdapter.java

```diff
@@ -53,7 +53,7 @@ public double getLeftStickX(){
     
     public double getRightStickX(){
         
-        return internalXboxController.getX(Hand.kRight);
+        return internalXboxController.getRawAxis(3);
```

Comment:
Either all stick getters should use the `getX`/`getY` calls or all should use the raw axis version. Mixing them makes this logic significantly harder to follow.